src/scripts.js

```diff
@@ -19,6 +19,8 @@ import { use } from 'chai';
 let recipeRepository;
 let randomUser;
 let user;
+let foundRecipe;
+let homeView = true;
```

Comment:
Never seen a global variable set to a boolean - would love to learn more about it!